web/src/components/case-details/common/AppearancesView.vue

```diff
@@ -2,7 +2,11 @@
   <v-row>
     <v-col cols="9" />
     <v-col>
-      <name-filter v-model="selectedAccused" :people="appearances" />
+      <name-filter
```

Comment:
```suggestion
      <NameFilter
```
To be consistent with other JASPER components